app/views/chargeback/_rates_tabs.html.haml

```diff
@@ -1,9 +1,9 @@
 #rates_tabs_div
-  -# Toplevel Tabbar
-  - if x_active_tree == :cb_rates_tree
-    - if x_node == "root"
-      = render :partial => "rates_list"
-    - elsif %w(Compute Storage).include?(x_node.split('-').last)
-      = render :partial => "cb_rates"
-    - else
-      = render :partial => "cb_rate_show"
+  - if x_node == "root"
+    = render :partial => "rates_list"
+  - elsif params[:pressed] && params[:pressed] != "chargeback_rates_delete"
```

Comment:
I'm missing here the test for `@edit`, would be more straightforward as we're using it on many places to distinguish between show/edit.